src/drawer/pages/Register/Register.tsx

```diff
@@ -1,47 +1,67 @@
+import { BoxButton } from '@yourssu/design-system-react';
+
 import { CategoryWithoutAll } from '@/drawer/components/CategoryWithoutAll/CategoryWithoutAll';
 import { Input } from '@/drawer/components/Input/Input';
 import { TextArea } from '@/drawer/components/TextArea/TextArea';
 import { WarningBox } from '@/drawer/components/WarningBox/WarningBox';
+import { LINK, RESISTER_URL } from '@/drawer/constants/link.constant';
+import { MOBILE_VIEW_WIDTH } from '@/drawer/constants/mobileview.constant';
 
-import { StyledContainer, StyledInputContainer } from './Register.style';
+import {
+  StyledContainer,
+  StyledImportText,
+  StyledInputContainer,
+  StyledRightContainer,
+} from './Register.style';
 
 export const Register = () => {
+  const validateLink = (name: string, value: string) => {
+    return !value.startsWith(RESISTER_URL[name as keyof typeof RESISTER_URL]);
+  };
+
   return (
     <StyledContainer>
       <StyledInputContainer>
-        <Input
-          maxLength={20}
-          title={'서비스 이름'}
-          description={'(최대 20자)'}
-          required={true}
-          isWarned={true}
-        />
-
-        <Input
-          width={79.625}
-          maxLength={20}
-          title={'간단한 설명'}
-          description={'(최대 20자)'}
-          required={true}
-          disabled={true}
-        />
+        <StyledRightContainer>
+          <StyledImportText>별표 표시 * 는 필수 입력란</StyledImportText>
+        </StyledRightContainer>
+
+        <Input maxLength={20} title={'서비스 이름'} description={'(최대 20자)'} required={true} />
+
+        <Input maxLength={20} title={'간단한 설명'} description={'(최대 20자)'} required={true} />
       </StyledInputContainer>
 
-      <TextArea title="내용" description="(최대 5000자)" required={true} maxLength={5000} />
+      <TextArea maxLength={5000} title={'내용'} description={'(최대 5000자)'} required={true} />
 
       <CategoryWithoutAll />
 
       <StyledInputContainer>
-        <Input maxLength={100} title={'웹 페이지 링크'} description={'(최대 100자)'} />
+        <StyledRightContainer>
+          <StyledImportText>
+            {window.innerWidth < MOBILE_VIEW_WIDTH
+              ? '웹 페이지, Google play, App store, Github\n링크 중 하나는 필수 기재 *'
+              : '웹 페이지, Google play, App store, Github 링크 중 하나는 필수 기재 *'}
+          </StyledImportText>
+        </StyledRightContainer>
 
-        <Input maxLength={100} title={'Google Play 링크'} description={'(최대 100자)'} />
-
-        <Input maxLength={100} title={'App Store 링크'} description={'(최대 100자)'} />
-
-        <Input maxLength={100} title={'GitHub 링크'} description={'(최대 100자)'} />
+        {LINK.map((link) => (
+          <Input
+            key={link.name}
+            maxLength={100}
+            title={link.title}
+            description={'(최대 100자)'}
+            validate={(value) => validateLink(link.name, value)}
+          />
+        ))}
```

Comment:
`map` 쓸 때 디스트럭처링하면 속성 바로 쓸 수 있아요
속성 2개 있는 객체에서는 취향따라 꼭 안 써도 될 거 같긴하지만 ㅎㅅㅎ

```suggestion
        {LINK.map(({ name, title }) => (
          <Input
            key={name}
            maxLength={100}
            title={title}
            description={'(최대 100자)'}
            validate={(value) => validateLink(name, value)}
          />
        ))}
```

Reply:
옹 저는 사실 link. 써주는 취향이긴 한데 ㅎㅅㅎ,,,,, 너무 길어질까용??

Reply:
근데 속성이 막..... 많고.... 이름도 길고..... 그러면 제가 이제 강경하게 디스트럭처링 밀었겠지만
지금은 그게 그거 같아요 사실...... 슬쩍 resolve